Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -153,7 +153,7 @@ public async Task<List<FleetAttributes>> DescribeFleetAttributes(ComputeType com
             }
             catch (Exception ex)
             {
-                Debug.Log(ex.Message);
+                Debug.LogError(ex.Message);
```

Comment:
What exactly is this meant to solve? All other methods in this file perform `Debug.Log` when an exception occurs.